test.js

```diff
@@ -12,3 +12,63 @@ test('sync', t => {
 	t.is(unusedFilename.sync('fixtures/unicorn.txt'), 'fixtures/unicorn (1).txt');
 	t.is(unusedFilename.sync('fixtures/rainbow.txt'), 'fixtures/rainbow (3).txt');
 });
+
+test('options.incrementer', async t => {
+	const testFile = 'fixtures/rainbow.txt';
+
+	const incrementerTest = async ({incrementer, error, message, result}) => {
+		if (error) {
+			t.throws(
+				() => unusedFilename.sync(testFile, {incrementer}),
+				error,
+				message
+			);
+
+			await t.throwsAsync(
+				async () => unusedFilename(testFile, {incrementer}),
```

Comment:
You are using this assertion incorrectly

```suggestion
				unusedFilename(testFile, {incrementer}),
```

Reply:
will do